core-services/release-controller/_repos/ocp-4.14-rhel86.repo

```diff
@@ -0,0 +1,59 @@
+[rhel-8.6-baseos]
+name = rhel-8.6-baseos
+baseurl = https://cdn.redhat.com/content/eus/rhel8/8.6/x86_64/baseos/os/
+enabled = 1
+gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted
+sslverify = false
+gpgcheck = 0
+# https://projects.engineering.redhat.com/browse/RCM-65421
+sslclientkey = /tmp/key/rh-cdn.pem
+sslclientcert = /tmp/key/rh-cdn.pem
+failovermethod = priority
+
+[rhel-8.6-appstream]
+name = rhel-8.6-appstream
+baseurl = https://cdn.redhat.com/content/eus/rhel8/8.6/x86_64/appstream/os/
+enabled = 1
+gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted
+sslverify = false
+gpgcheck = 0
+# https://projects.engineering.redhat.com/browse/RCM-65421
+sslclientkey = /tmp/key/rh-cdn.pem
+sslclientcert = /tmp/key/rh-cdn.pem
+failovermethod = priority
+
+[rhel-8.6-fast-datapath]
+name = rhel-8.6-fast-datapath
+baseurl = https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/fast-datapath/os/
+enabled = 1
+gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted
+sslverify = false
+gpgcheck = 0
+# https://projects.engineering.redhat.com/browse/RCM-65421
+sslclientkey = /tmp/key/rh-cdn.pem
+sslclientcert = /tmp/key/rh-cdn.pem
+failovermethod = priority
+
+[rhel-8.6-nfv]
+name = rhel-8.6-nfv
+baseurl = https://cdn.redhat.com/content/tus/rhel8/8.6/x86_64/nfv/os/
+enabled = 1
+gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted
+sslverify = false
+gpgcheck = 0
+# https://projects.engineering.redhat.com/browse/RCM-65421
+sslclientkey = /tmp/key/rh-cdn.pem
+sslclientcert = /tmp/key/rh-cdn.pem
+failovermethod = priority
+
+[rhel-8.6-server-ose-4.13]
```

Comment:
@jmguzik @psalajova Shouldn't this and the following line be bumped to `4.14` too?

Reply:
in my opinion, yes